packages/amplify-frontend-android/lib/amplify-config-helper.js

```diff
@@ -25,6 +26,37 @@ function constructAuth(metadata, amplifyConfig, awsConfig) {
   }
 }
 
+function constructNotifications(metadata, amplifyConfig) {
```

Comment:
since we can't type this, can we add a JSDoc that describe the shape on the inputs and outputs? (same applies to the functions for the other frontend types as well)

Can we also get unit tests for these functions?